homeassistant/components/sensor/sensehat.py

```diff
@@ -0,0 +1,139 @@
+"""
+Support for Sense HAT sensors.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/
+"""
+import os
+import logging
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/RPi-Distro/python-sense-hat'
+                '/archive/master.zip#sense-hat==2.2.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_cpu_temp():
```

Comment:
The reason that I calculate the temperature is that the value that sensehat gives as a temperature is affected by CPU temperature so it is wrong and high... because of that I did it in the way that the temperature would be as precise as possible.